Adafruit_DAP_nRF5x.cpp

```diff
@@ -268,7 +268,7 @@ bool Adafruit_DAP_nRF5x::program(uint32_t addr, const uint8_t *buf,
 
   while (count) {
     uint8_t data[CHUNK_SIZE];
-    uint32_t bytes = min(count, CHUNK_SIZE);
+    uint32_t bytes = min(count, (size_t) CHUNK_SIZE);
```

Comment:
the cast should be `uint32_t` instead of size_t, though they are the same. both `count` and the result `bytes` are uint32_t. It will make more sense.

Reply:
Just did, my initial thought was to make it like elsewhere in the library, but you are right. This makes more sense.